terms-of-service.md

```diff
@@ -1,3 +1,90 @@
-# Terms
+# Terms of Service
+
+---
+title: Terms
```

Comment:
@lex124 This is specific to the website pages. We shall remove these lines from here.